setup.py

```diff
@@ -56,6 +56,9 @@
     'nc_nwcsaf_msg': ['netCDF4 >= 1.1.8'],
     'sar_c': ['python-geotiepoints >= 1.1.7', 'rasterio', 'rioxarray'],
     'abi_l1b': ['h5netcdf'],
+    'seviri_l1b_hrit': ['pyorbital >= 1.3.1'],
+    'seviri_l1b_native': ['pyorbital >= 1.3.1'],
+    'seviri_l1b_nc': ['pyorbital >= 1.3.1'],
```

Comment:
That reader needs netCDF4 installed also.

Reply:
Great, I added netCDF4 as well!